src/main/java/se/sundsvall/incident/integration/db/entity/IncidentEntity.java

```diff
@@ -24,9 +24,7 @@
 @Builder(setterPrefix = "with")
 @Getter
 @Setter
-public class IncidentEntity implements Serializable {
-
-	private static final long serialVersionUID = -8817006765529811100L;
+public class IncidentEntity {
 
 	@Id
 	@Column(name = "IncidentId")
```

Comment:
Upper camel case in the database column?
(Isn't snake case theconvention? The other entities in this PR are using snake_case)